### DISCUSSION TITLE
Dwains Dashboard not visible in integrations

### DISCUSSION THREAD

**Discussion by multispeed** [dwainscheeren/dwains-lovelace-dashboard#305] [Q&A]
Hi!

So tonight i downloaded Dwains Dashboard and placed the folder dwains_dashboard in/config/custom_components/dwains_dashboard i then rebooted and tried to add it by going to Conmfiguration - Integrations and then add it by clicking the +

But the strange thing is Dwains Dashboard just isn't there..

What am i missing here ?

**Comment by multispeed** [comment]
Al of a sudden the next day it is visible in entities. 

So it's solved

**Comment by dwainscheeren** [comment]
Hi,

I am closing the old Dwains Dashboard 3.x discussions as part of the DD3 cleanup.

DD3 is now in maintenance/community mode and is slowly being phased out. Active development is focused on Dwains Dashboard Next, the newest dashboard project:

https://github.com/dwainscheeren/dwains-dashboard-next

For chat, community help and discussion around DD3/DD Next, please join the Discord:

https://discord.gg/7yt64uX

If you still have a reproducible DD3 bug on the latest 3.9.x release, please open a new issue with current details. For new feature ideas, please try Dwains Dashboard Next.
